WebStoreApplication/Views/Home/Login.cshtml

```diff
@@ -3,18 +3,87 @@
     ViewData["Title"] = "Login";
 }
 
-<head>
-    <meta charset="utf-8" />
-    <link rel="stylesheet" href="C:\Users\liebenbergd03\Downloads\style.css" />
-</head>
-<body class="form" id="login">
+<form action="action_page.php" method="post">
     <h1>Login</h1>
-    <div>
-        <form class="login-form">
-            <input type="text" placeholder="Username" />
-            <input type="password" placeholder="Password" />
-            <button id="LoginButton">login</button>
-        </form>
+    <div class="container">
+        <label for="uname"><b>Username</b></label>
+        <input type="text" placeholder="Enter Username" name="uname" required>
+
+        <label for="psw"><b>Password</b></label>
+        <input type="password" placeholder="Enter Password" name="psw" required>
+
+        <button class="btn-dark">Login</button>
+        
+    </div>
+
+    <div class="container" style="background-color:#f1f1f1">
+        <button type="button" class="cancelbtn" onclick="@("window.location.href='" + @Url.Action("Index", "Home") + "'");">Cancel</button>
     </div>
-</body>
+</form>
+
+<style>
```

Comment:
Please use default bootstrap styling. if you need custom styling then please import the stylesheet as apposed to having it at the bottom of the html